modules/apps/simplicity-theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+
+<#include init />
+
+<html class="${root_css_class}" dir="<@liferay.language key="lang.dir" />" lang="${w3c_language_id}">
+
+<head>
+	<title>${the_title} - ${company_name}</title>
+
+	<meta content="initial-scale=1.0, width=device-width" name="viewport" />
+
+	<@liferay_util["include"] page=top_head_include />
+</head>
+
+<body class="${css_class}">
+
+<@liferay_ui["quick-access"] contentId="#main-content" />
+
+<@liferay_util["include"] page=body_top_include />
+
+<@liferay.control_menu />
+
+<div class="container-fluid" id="wrapper">
+	<header id="banner" role="banner">
+		<div class="custom-header">
+			<div class="header-media">
+				<img alt='<@liferay.language key="main-theme-image" />' class="simplicity-media" src="${themeDisplay.getPathThemeImages()}/${site_header_image}" />
+			</div>
+
+			<div class="site-branding">
+				<div class="wrap">
+					<h1 class="site-title">
+						<#if show_site_name>
+							<span class="site-name" title="<@liferay.language_format arguments="${site_name}" key="go-to-x" />">
+								${site_name}
+							</span>
+						</#if>
+					</h1>
+					<div class="site-description">${site_description}</div>
+					<div class="sign-in-link">
+						<#if !is_signed_in>
+							<a data-redirect="${is_login_redirect_required?string}" href="${sign_in_url}" id="sign-in" rel="nofollow">${sign_in_text}</a>
+						</#if>
+					</div>
```

Comment:
We should flip the logic so the ```<#if />``` wraps the ```<div />``` otherwise if it is ` false`, we have a blank ```<div />``` that is being output.